partners/yieldyak

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
The format for the JSON submission is not correct, it should look something like this
[here](https://github.com/Luna-Exchange/avalanche-rewards-projects?tab=readme-ov-file#nft-mission-dokyo)